mongodbatlas/resource_mongodbatlas_search_index.go

```diff
@@ -272,6 +293,10 @@ func resourceMongoDBAtlasSearchIndexRead(ctx context.Context, d *schema.Resource
 		return diag.Errorf("error setting `index_id` for search index (%s): %s", d.Id(), err)
 	}
 
+	if err := d.Set("type", searchIndex.Type); err != nil {
```

Comment:
Does the API always responds with a value for the type field? Would consider marking as computed or defining a default value in the schema to avoid inconsistencies in the state.

Reply:
if the index was created withouth type, it won't return type.   API responds same as in creation

Reply:
suggest to have this question/answer represented with a test, not necessarily blocking this PR (unless it's trivial creating it).